schema.ts

```diff
@@ -870,6 +870,10 @@ export interface PropertyIdentifierDetails {
    * If set to '', the label will be omitted.
    */
   label?: string;
+  /**
+   * An optional placeholder value, in case the property value is null or undefined.
```

Comment:
Would probably just be really explicit in the comment, e.g. "which will be rendered when the property value is null or undefined."

And then just to be clear, if the property value is empty/falsey, will the placeholder be used or not?

Reply:
Yeah that's tricky. I feel if we get one of these cases:
- undefined
- null
- []
- ""
Then it should show the placeholder.

Reply:
Yeah, I leave it up to you all to figure out what makes the most sense. Would just be very explicit in the documentation about which cases apply here.